### PR TITLE
Copy build to root, update to node 14

### PR DESCRIPTION
**Pull request by faction23**
## What does this do/fix?

[Issue](https://moderntribe.atlassian.net/browse/SQONE-590)

Copies the build system to root. I left the theme one intact for now (let me know if you want me to delete it now Nick), as it can run from there too. In the next month, we'll want to move much of this functionality to an npm package so build is managed in its own repo and can be used in any directory easily. 

There is one file that has to change in theme right now for the build to work in root, and thats the font resolution for the icon kit. Its the only piece i'll have to solve in a new way. For now, all tasks run from root, tests etc. 

I removed the husky pre commit eslint check, as it was causing me issues, but left the php side of it. We can chat about what the goal with husky is there, since we have eslinting baked into our build.

@ryanurban @dpellenwood would love to chat sometime in what i am now seeing in the output css, seems a little wonky

### CODE REVIEW

File: wp-content/themes/core/assets/css/src/theme/icons/icons.pcss [hunk 1]
```diff
@@ -9,10 +9,10 @@
 @font-face {
 	font-family: 'core-icons';
 	src:
-		resolve('assets/fonts/theme/icons-core/core-icons.woff2?k9lxjz') format('woff2'),
-		resolve('assets/fonts/theme/icons-core/core-icons.ttf?k9lxjz') format('truetype'),
-		resolve('assets/fonts/theme/icons-core/core-icons.woff?k9lxjz') format('woff'),
-		resolve('assets/fonts/theme/icons-core/core-icons.svg?k9lxjz#core-icons') format('svg');
+		resolve('wp-content/themes/core/assets/fonts/theme/icons-core/core-icons.woff2?k9lxjz') format('woff2'),
```
Comment (dpellenwood) [inline wp-content/themes/core/assets/css/src/theme/icons/icons.pcss:12]: With the `postcss-assets` changes above, you should be able to revert this, I think.

**Review by dpellenwood — COMMENTED** [review]
Couple questions and (hopefully) a solution for the postcss-assets/resolve() issue. Looks great. Thank you, Sam!

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-12.13.1
\ No newline at end of file
+14.15.0
```
[the remark this answers is not in the document]
Comment (faction23) [reply .nvmrc:1]: we have some projects here on 14.15.0. i tend to sue one lts per year across tribe. every november we bump to the next fixed version.

File: babel.config.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+module.exports = {
+	presets: [
+		'@babel/preset-react',
+		[
+			'@babel/preset-env',
+			{
+				targets: 'last 2 versions, safari >= 13, ios >= 13, android >= 5.1, not ie <= 11, not ie_mob <= 11, not bb <= 10, not samsung 4, not op_mob <= 12.1',
```
[the remark this answers is not in the document]
Comment (faction23) [reply babel.config.js:7]: sure, just chitterlings

File: gulp-tasks/postcss.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+			pkg.square1.paths.core_theme_img,
+			pkg.square1.paths.core_admin_img,
+		],
+	} ),
+	require( 'postcss-preset-env' )( { stage: 0, autoprefixer: { grid: true }, features: { 'focus-visible-pseudo-class': false, 'focus-within-pseudo-class': false } } ),
+	require( 'postcss-calc' ),
+];
+
+const compilePlugins = sharedPlugins.concat( [
+	require( 'postcss-assets' )( { baseUrl: pkg.square1.paths.core_theme_postcss_assets_base_url } ),
```
[the remark this answers is not in the document]
Comment (faction23) [reply gulp-tasks/postcss.js]: Done!

File: wp-content/themes/core/assets/css/src/theme/icons/icons.pcss (hunk 1 above)
Comment (faction23) [reply wp-content/themes/core/assets/css/src/theme/icons/icons.pcss]: Done!

**Review by dpellenwood — APPROVED** [review]
Thanks for tackling this Sam! :shipit:

**Comment by nickpelton** [comment]
Did a quick test and hit a nvmrc mismatch. Fixed.

Also removed SO commands, not needed here anymore.

**Review by nickpelton — APPROVED** [review]
🚀

**Pull request merged**